blog/authors.yml

```diff
@@ -1,25 +1,53 @@
-yangshun:
-  name: Yangshun Tay
-  title: Ex-Meta Staff Engineer, Co-founder GreatFrontEnd
-  url: https://linkedin.com/in/yangshun
-  image_url: https://github.com/yangshun.png
+castrojo:
+  name: Jorge O. Castro
   page: true
+  title: Developer Relations
+  url: https://github.com/castrojo
+  image_url: https://github.com/castrojo.png
+  email: jorge.castro@gmail.com
   socials:
-    x: yangshunz
-    linkedin: yangshun
-    github: yangshun
-    newsletter: https://www.greatfrontend.com
-
-slorber:
-  name: Sébastien Lorber
-  title: Docusaurus maintainer
-  url: https://sebastienlorber.com
-  image_url: https://github.com/slorber.png
-  page:
-    # customize the url of the author page at /blog/authors/<permalink>
-    permalink: '/all-sebastien-lorber-articles'
-  socials:
-    x: sebastienlorber
-    linkedin: sebastienlorber
-    github: slorber
-    newsletter: https://thisweekinreact.com
+    github: castrojo
+    linkedin: jorge-castro2112
+    newsletter: https://ypsidanger.com
+caniszczyk:
+  name: Chris Aniszczyk
+  page: true
+  title: fixme
+  url: https://github.com/caniszczyk
+  image_url: https://github.com/caniszczyk.png
+  email: todo
+idvoretskyi:
+  name: Ihor Dvoretskyi
+  page: true
+  title: fixme
+  url: https://github.com/idvoretskyi
+  image_url: https://github.com/idvoretskyi.png
+  email: todo
+jeefy:
+  name: Jeffrey Sica
+  page: true
+  title: fixme
+  url: https://github.com/jeefy
+  image_url: https://github.com/jeefy.png
+  email: fixme
+krook:
+  name: Daniel Krook
+  page: true
+  title: Senior Director of Developer Experience
+  url: https://github.com/krook
+  image_url: https://github.com/krook.png
+  email: fixme
```

Comment:
krook:
  name: Daniel Krook
  page: true
  title: Senior Director of Developer Experience
  url: https://krook.net/
  image_url: https://github.com/krook.png
  email: dkrook@linuxfoundation.org
  socials:
    bluesky: krook.net
    github: krook
    linkedin: krook